test/app-luatest/msgpack_test.lua

```diff
@@ -445,3 +445,67 @@ g.test_object_misc = function()
     t.assert_not(msgpack.is_object(it))
     t.assert_not(msgpack.is_object({mp}))
 end
+
+-- Checks that `msgpack.object:get` method works correctly.
+g.test_object_get = function()
+    local array = {nil, true, 1, '2', 3.0, {1, 2, 3}, {a = 1, b = 2, c = 3}, 'decode', 'iterator', 'get'}
```

Comment:
This isn't an array from Lua point of view - it's a map, because of the nil. The code testing arrays isn't covered at all: https://coveralls.io/jobs/110054477/source_files/28263573271#L819

Reply:
> it's a map, because of the nil

Didn't know about that, thanks!

> The code testing arrays isn't covered at all: [coveralls.io/jobs/110054477/source_files/28263573271#L819](https://coveralls.io/jobs/110054477/source_files/28263573271#L819)

Thanks for pointing this out, I will also check coverage next time when submitting a patch.